### DISCUSSION TITLE
The Upcoming knitr v1.44: More Compatible with Quarto, and Locking `opts_current` | /en/2023/09/knitr-1-44/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1511] [Comments]
After coming back from vacation, I mainly spent my time on the knitr package maintenance. I will make a new release soon, and want to quickly highlight a few things in this post. For the full list of &hellip;

https://yihui.org/en/2023/09/knitr-1-44/

**Comment by NetZissou** [comment]
Hi Yihui, are you planning to attend this year's Posit Conf?

**Reply by yihui** [reply]
No, I won't be there, but I may lurk on Discord or other virtual platforms.

**Reply by NetZissou** [reply]
Thanks, chief! I am attending in person for the first time. I hope you are doing well!

**Reply by yihui** [reply]
Thank you! Have fun in Chicago! It's a short flight from where I live, but I have pretty much stopped attending conferences in person since COVID (not because of COVID, but just that I've become much less interested in traveling).
